### ISSUE TITLE
Need: logo and favicon

### ISSUE THREAD

**Issue by getify** [getify/youperiod.app#32]
Related to #2.

I kinda like the logo I originally designed for this, as can be seen in the attached image. But I'm not deeply attached to it. If someone comes up with some good variations on logo, we can certainly discuss them here.

<img width="185" alt="youperiod-logo-screenshot" src="https://user-images.githubusercontent.com/150330/177574622-1212f577-5299-4423-acc2-170ea2ca6cb5.png">

More importantly, whatever logo we pick, we need:

1. SVG of it (with font glyphs changed to curves)
2. PNG of it in a couple of general sizes (small ~100px, medium ~400px, large ~1000px) on both transparent, light, and dark backgrounds
3. Favicon that is based on, or recognizable as related to, the logo. I had envisioned something like a red circle with a "Y" in the middle of it, as the favicon. But certainly open to input. The favicon should be square and have 16x16, 24x24, 32x32, and 64x64 resolutions embedded into it. I use an app (windows) called Icofx (v3) for making my own favicons like this.

**Comment by detronetdip** [comment]
Do you want to put `YOU.` in the logo or it can be `U.` and do you want to use red and black only?

**Comment by getify** [comment]
I definitely want "YOU" (or "You" depending on the typography) and an an enlarged dot/period. I think the red period communicates some of the meaning of the app, subtly, but if it's a little too harsh or "on the nose", I am not deeply attached to those colors.

I do think it's important that the "YOU" and the dot be distinctly different colors, so the dot is not easily overlooked. I want the app to be pronounced (verbally and sub-vocally in your head), and described in text, as "You Period" rather than just "You".

**Comment by getify** [comment]
Side note: I spent quite a bit of time trying to find an appropriate unicode character to use for the dot/period that was larger but still bottom-aligned. I found several characters, but they were all middle-aligned vertically.

The reason that matters is, both from a semantic HTML/text perspective, I'd like there to be a different character than just "." in the text. Also, annoyingly, we can't really control the typography of the site's title as it appears in the browser tab, nor as it appears in search results lists, etc. So it'd be ideal if there just was a "bold period" or "strong period" or something like that in unicode. But I couldn't find it.

**Comment by getify** [comment]
I don't know whether a blood drop will be a little too .... forward .... too literal. Depends on the design, but I'm not sure someone will see a blood drop and think the word "period", rather than "you blood" or "you drop".

**Comment by detronetdip** [comment]
No I was just thinking about that because it's a bit related with  what we are making.

**Comment by getify** [comment]
It might work to have a blood drop in the design somehow... not sure. Or it might confuse the user who might assume this is an app for tracking blood draws or blood sugar measurements.

**Comment by aditiranka27** [comment]
I agree with @getify, a blood drop would be too much and too literal (imo I would not want see any blood drop images in the app). I would suggest with sticking to "YOU." for the logo design and keeping the "." only to Red color. (the red color should only be used to depict the "." in the app. For the overall design and color schema, it is being discussed in #2, keeping cool/calm tones with blue/green or purple/creamy white or a mix of pastel shades.

Also, a blood drop image should be used only for a feature depicting the blood flow during the menstrual cycle.

**Comment by detronetdip** [comment]
![1](https://user-images.githubusercontent.com/64276906/177598429-9b96b724-ec62-45e8-b1e1-fe68cd682edb.png)

@getify  Can I use this font for the logo?

**Comment by getify** [comment]
I'd like us to play around a bit with the font weight... that font seems pretty light, and the smaller it is, it may be a little harder to read or less contrast. Also, perhaps this is a case where the lighter the font, maybe the lowercase "ou" looks better? Not sure.

Definitely think the period needs to be much more prominent in size compared to the "u" it's next you, some of which depends on the uppercase vs lowercase.

**Comment by detronetdip** [comment]
@getify  I have added a few `You.` with different font style in the `Brainstorming` project.

**Comment by getify** [comment]
I messaged with the logo in the figma project, and also toyed with a similar ideas for the favicon. Thoughts here?

<img width="398" alt="youperiod-logo-favicon-idea" src="https://user-images.githubusercontent.com/150330/178387382-66c80a95-4fbb-46c0-bb82-4225babd3f1d.png">

So there would be a white circle (no border) behind the "u." which is the same font from the full logo. The period is overlayed on the "u" in the favicon, to keep things smaller. Not sure if that would also work in the main logo or not.

**Comment by getify** [comment]
I don't want to use "u" for the logo. fine with it in the favicon.
